Source/Text/SvgTextBase.cs

```diff
@@ -257,56 +257,10 @@ public override RectangleF Bounds
             }
         }
 
-        /// <summary>
-        /// Renders the <see cref="SvgElement"/> and contents to the specified <see cref="Graphics"/> object.
-        /// </summary>
-        /// <param name="renderer">The <see cref="ISvgRenderer"/> object to render to.</param>
-        /// <remarks>Necessary to make sure that any internal tspan elements get rendered as well</remarks>
-        protected override void Render(ISvgRenderer renderer)

```

Comment:
Main purpose of this PR is to erase this method.

Reply:
Hm, I don't understand that initial comment about the tspan elements, so I also don't understand how you got rid of this problem. Can you elaborate on this?

Reply:
@mrbean-bremen 

The intent of this comment is to call `RenderChildren` method.
This has been moved to `RenderFillAndStroke` method.
I think this PR will be necessary in the future when using `filter` in text.

Reply:
Ok, thanks - I get it!